### DISCUSSION TITLE
i cannot seem to find the persistent##.bak file

### DISCUSSION THREAD

**Discussion by chapper79** [Monika-After-Story/MonikaModDev#8500] [QA - tech support + general]
i am trying to restore an older persistent file but I am unable to find the persistent##.bak like the instructions in https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-need-to-restore-an-older-persistent seem to believe I have. Sorry if this is a stupid issue, I am not too great with tech despite growing up with this stuff.

**Comment by ThePotatoGuy** [comment]
The instructions for finding the persistent are [here](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-want-to-back-up-my-persistent).
